compiler/codegen/OMRLinkage.hpp

```diff
@@ -75,27 +75,17 @@ class OMR_EXTENSIBLE Linkage
 
    virtual void setParameterLinkageRegisterIndex(TR::ResolvedMethodSymbol *method, List<TR::ParameterSymbol> &parm)
```

Comment:
In the spirit of other functions in this base class, these might be made pure virtual functions.  However, I wasn't about to start pulling that thread in this basic cleanup.  That can be explored as part of #2082 eventually.